src/flights.py

```diff
@@ -13,4 +13,75 @@
 
 When this code is completed, you'll import it into this file and write your Flights program, 
 which you should be able to do on your own.
-'''
\ No newline at end of file
+'''
+
+from tables import *
+import os
+import operator
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
You seem to have some leftover code here that was moved into the `get_data()` function